shared/api.ts

```diff
@@ -44,11 +54,8 @@ export class Api {
 
   private async getAccessToken(): Promise<string | undefined> {
     // renew
-    if (this.accessToken == null || this.expires == null || this.expires <= new Date()) {
-      const result = await axios.post<{ accessToken: string }>(`${this.apiUrl}/auth/sign-in`, {
-        address: Config.api.address,
-        signature: Config.api.signature,
-      });
+    if ((this.accessToken == null || this.expires == null || this.expires <= new Date()) && !this.authenticationInfo) {
```

Comment:
Are you sure this is doing the right thing?

Reply:
at first yes, if you ask like this... no... will rewrite it